src/main/java/serverutils/lib/data/Universe.java

```diff
@@ -228,6 +233,24 @@ public Universe(MinecraftServer s) {
         taskQueue = new ArrayList<>();
     }
 
+    private void cleanup() {
+        server = null;
+        world = null;
+        players = null;
+        vanishedPlayers = null;
+        teams = null;
+        teamMap = null;
+        noneTeam = null;
+        uuid = null;
+        fakePlayerTeam = null;
+        fakePlayer = null;
+        taskList = null;
+        taskQueue = null;
+        ticks = null;
+        dataFolder = null;
+        latModFolder = null;
```

Comment:
Won't these all still be null if you quit and rejoin the world?

Reply:
They will still all be null if you do that. But it doesn't matter because that instance of Universe will not be reachable by the mod, it creates a new i stance - that will be a different one - when you join the world.